doc/WebSite/Feature-Management.md

```diff
@@ -29,20 +29,20 @@ Can be "true" or "false". This type of a feature can be **enabled** or
 
 #### Value Feature
 
-Can be an **arbitrary value**. While it's stored and retrieved a string,
+Can be an **arbitrary value**. While it's stored and retrieved as a string,
 numbers also can be stored as strings.
 
 For example, our application may be a task management application and we
 may have a limit for creating tasks in a month. Say that we have two
-different edition/package; one allows creating 1,000 tasks per month,
-while other allows creating 5,000 tasks per month. So, this feature
-should be stored as value, not simply true/false.
+different edition/package; one allows for creating 1,000 tasks per month,
+while the other allows for creating 5,000 tasks per month. This feature
+should be stored as a value, not simply as true or false.
 
 ### Defining Features
 
-A feature should be defined before checking. A
+A feature should be defined before it is checked. A
 [module](/Pages/Documents/Module-System) can define it's own features by
```

Comment:
`its`

Reply:
it's

Reply:
`It's` is an abbreviation of `it is` → `A module can define it is own features` doesn't read right.

Reference: https://en.oxforddictionaries.com/usage/its-or-it-s

Reply:
Nevermind, I see you are referencing line 44